mteb/tasks/BitextMining/multilingual/RomaTalesBitextMining.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskBitextMining
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class RomaTalesBitextMining(AbsTaskBitextMining):
+    metadata = TaskMetadata(
+        name="RomaTalesBitextMining",
+        dataset={
+            "path": "kardosdrur/roma-tales",
+            "revision": "f4394dbca6845743cd33eba77431767b232ef489",
+        },
+        description="Parallel corpus of Roma Tales in Lovari with Hungarian translations.",
+        reference="https://idoc.pub/documents/idocpub-zpnxm9g35ylv",
+        type="BitextMining",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["rom-Latn", "hun-Latn"],
```

Comment:
I think that `eval_langs` for BitextMinnig tasks should be language pairs. 
```suggestion
        eval_langs={
             "ro-hu": ["rom-Latn", "hun-Latn"]
        }
```

Reply:
I believe I copied most of it over from another task (maybe Norwegian courts), so this might be a more wide-spread issue. It would also probably make sense to extend the validation to catch this. Should I file an issue?

Reply:
For the CrossLingualTask inheritance, I think it should be done even for [NorwegianCourts](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/BitextMining/multilingual/NorwegianCourtsBitextMining.py). This should allow to run bitext mining tasks by passing a language pair to the evaluator and select the right language pair subset.
Although for the language mapping I'm not very sure of what's the correct way of doing it. [BUCC](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/BitextMining/multilingual/BUCCBitextMining.py) or [Tatoeba](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/BitextMining/multilingual/TatoebaBitextMining.py) are different.
Not sure which one is correct since we did a refactoring on mteb and changed the language mapping. @KennethEnevoldsen an opinion on this?

Reply:
I think you're right. But I think that the correct way of doing it is as in [DiaBla](https://github.com/embeddings-benchmark/mteb/blob/main/mteb%2Ftasks%2FBitextMining%2Fmultilingual%2FDiaBLaBitextMining.py), we created the language pairs in the data transform function. Lmt if it's not clear!

Reply:
I'll take a look!!

Reply:
I think I did it, can you take a look?

Reply:
I think the previous approach work just fine, the language mapping is really only needed for multilingual datasets. However, agree with @imenelydiaker, that in this case this formulation is more consistent.